test/small_tests.js

```diff
@@ -632,18 +630,54 @@ students
      dname
     `).getDDL();
 
-    //console.log(output);
     assert( "0 < output.indexOf('number default on null to_number(sys_guid()')" );
 
-        
-}    
+    // https://github.com/oracle/quicksql/issues/51
+    output = new quicksql(`boolvalues
+    is_legal
+    finished_yn
+    ok   bool
+    yes  boolean
+    #boolean:native`).getDDL();
+
+    assert( "0 < output.indexOf('is_legal       boolean,')" );
+    assert( "0 < output.indexOf('finished_yn    boolean,')" );
+    assert( "0 < output.indexOf('ok             boolean,')" );
+    assert( "0 < output.indexOf('yes            boolean')" );
+
+    // https://github.com/oracle/quicksql/issues/51
+    output = new quicksql(`boolvalues
+        ok   bool
+        #db:"23c"`).getDDL();
+    assert( "0 < output.indexOf('ok    boolean')" );
+
+    // https://github.com/oracle/quicksql/issues/55
+    output = new quicksql(`escape /insert 1
+    financial_year /check '23/24', \`'24/25'\`
+    surname vc60 /check 'O''Hara', q'{O'Tool}'  
+    start_date /check  \`to_date('01-APR-2025','DD-MON-YYYY')\``).getDDL();
+    assert( "0 < output.indexOf(\"check (financial_year in ('23/24','24/25')),\")" );
+    assert( "0 < output.indexOf(\"check (surname in ('O''Hara',q'{O'Tool}')),\")" );
+    assert( "0 < output.indexOf(\"check (start_date in (to_date('01-APR-2025','DD-MON-YYYY')))\")" );
+    assert( "output.indexOf(\"''24/25''\") < 0" );
+    assert( "output.indexOf(\"q''{O''Tool}''\") < 0" );
+    assert( "output.indexOf(\"to_date(''01-APR-2025'',''DD-MON-YYYY'')\") < 0" );
+  
+    output = new quicksql(`departments /insert 1
+    name /nn
+    # settings = {"prefix":"test"}`).getDDL();
+    //console.log(output);
+    assert( "0 < output.indexOf(\"insert into test_departments (\")" );
+
+} 
+
```

Comment:
Please add a regression test for this scenario:


```
departments /insert 4
  name /nn


# settings = {"prefix":"test"}
```

Where prefix setting is breaking inserts

Reply:
Not sure what you are requesting. At line 666:
`
    output = new quicksql(`departments /insert 1
    name /nn
    # settings = {"prefix":"test"}`).getDDL();
    //console.log(output);
    assert( "0 < output.indexOf(\"insert into test_departments (\")" );
`
is the test that does that.

Reply:
Apologies Vadim, I did not see it :)